app/components/point.jsx

```diff
@@ -4,6 +4,7 @@
 import React, { useState } from 'react'
```

Comment:
please add a link to this issue as a comment right below the link to the original point component issue
The case of linking an issue to a component that is already linked to an issue is new, so not mentioned in the README coding guidelines, but I see this be useful in the future - to the next person.